server/models/review-request.js

```diff
@@ -42,4 +42,34 @@ const reviewSchema = new mongoose.Schema({
   },
 });
 
+// request model
+const requestSchema = new mongoose.Schema({
+  // References user owner
+  userOwner: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+  },
+
+  selectedReviewer: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: "User",
+  },
+
+  // Needed for timeout eventually
+  requestCreateDate: {
+    type: Date,
+  },
+  reviewersDeclined: [
+    {
+      type: String,
+    },
+  ],
+  status: {
+    type: String,
+  },
```

Comment:
If status is going to be a finite state machine, it would make sense to make it enum and add transition methods to the requestSchema prototype

Reply:
Apologies can you elaborate more on instances?